eggplant/membership/models/account.py

```diff
@@ -28,7 +30,7 @@ def __str__(self):
 
 
 class Account(models.Model):
-    number = models.PositiveSmallIntegerField()
+    number = models.AutoField(primary_key=True)
```

Comment:
I think it's a little tricky that we have essentially renamed 'id' to 'number' -- couldn't we just remove it then? :)